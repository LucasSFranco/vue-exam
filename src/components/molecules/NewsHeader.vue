<template>
  <header class="header">
    <a
      target="_blank"
      rel="noreferrer"
      href="https://www.nytimes.com/"
    >
      <img
        src="@/assets/logo.svg"
        alt="The New York Times"
      />
    </a>
  </header>
</template>

<script>

export default {
  name: 'NewsHeader',
}

</script>

<style lang="scss" scoped>

  .header {
    display: flex;
    align-items: center;
    justify-content: center;

    border-bottom: 1px solid var(--accent-secondary);
    padding: .75rem 0;

    a {
      width: 185px;
      height: 25px;

      display: flex;
    }
  }

</style>
