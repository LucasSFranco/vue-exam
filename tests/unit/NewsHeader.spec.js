import { shallowMount } from '@vue/test-utils'

import NewsHeader from '@/components/molecules/NewsHeader.vue'

describe('News Header', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(NewsHeader)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
