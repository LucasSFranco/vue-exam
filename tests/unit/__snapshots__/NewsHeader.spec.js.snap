// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News Header matches snapshot 1`] = `<header class="news header"><a target="_blank" rel="noreferrer" href="https://www.nytimes.com/" data-test="nyt-logo"><img src="" alt="The New York Times"></a></header>`;
